src/setup/cortx_deploy

```diff
@@ -110,26 +113,60 @@ node_id=`cat /etc/machine-id`
 cluster_id=$(conf_get $(yaml_config $CLUSTER_INFO) "cluster>id")
 log $INFO "Starting provisioning on node $node_id which is part of cluster $cluster_id."
 
-# Apply Cluster Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE' $FORCE_PROVISIONING"
-cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE" $FORCE_PROVISIONING;
-
-# Apply Common Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c '$CORTX_CONFSTORE' $FORCE_PROVISIONING"
-cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c "$CORTX_CONFSTORE" $FORCE_PROVISIONING;
-
-# Validate config
-if ! [ -z $VALIDATE_CONF ];
-then
-    log $INFO "cortx_setup config validate -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE'"
-    cortx_setup config validate -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE" || exit 1
-    log $INFO "cortx_setup config validate -f $(yaml_config $CONFIG_INFO) -c '$CORTX_CONFSTORE'"
-    cortx_setup config validate -f $(yaml_config $CONFIG_INFO) -c "$CORTX_CONFSTORE" || exit 1
+# Check current image version
+current_image_version=$(conf_get $CORTX_CONFSTORE "cortx>common>release>version")
+# if current version is not available in confstore
+if [ $current_image_version = "[null]" ]; then
+    DEPLOY_FLAG=true
+else
+    # Below can be used when subversioning is considered.
+    #current_primary_ver=$(awk -F. '{print $1 "." $2 "." $3}' <<< $current_image_version)
+    current_version=$(awk -F. '{print $4}' <<< $current_image_version)
 fi
 
-# Bootstrap Cluster
-log $INFO "cortx_setup cluster bootstrap -c '$CORTX_CONFSTORE' $FORCE_PROVISIONING"
-cortx_setup cluster bootstrap -c "$CORTX_CONFSTORE" $FORCE_PROVISIONING;
+update_image_version=$(sed -e 's/[][]//g' -e 's/"//g' <<< $(conf_get $RELEASE_INFO "VERSION"))
```

Comment:
This is not update_image_version. This is "current_image_version" 

We need to compare current_image_version with current_installed_version.

Reply:
Done